Source/Core/Core/CoreTiming.cpp

```diff
@@ -50,7 +50,7 @@ static Common::FifoQueue<BaseEvent, false> tsQueue;
 // event pools
 static Event *eventPool = nullptr;
 
-static float lastOCFactor;
+float lastOCFactor;
```

Comment:
My impression is that he stuck to the predominant style inside that file; since the other members don't have those prefixes yet. I'd prefer to either keep them consistent (as is now) or change all of them (which might be out of scope of this PR).